packages/core/docs/commercetools/configuration.md

```diff
@@ -18,7 +18,33 @@ Commercetools configuration is located in two places:
 }]
 ```
 
-- `useNuxtI18nConfig` - when this property is set to true, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. When set to false, `i18n` config should be declared directly inside this package configuration. You can read more about `i18n` config in Vue Storefront [here](../advanced/internationalization.md)
+- `useNuxtI18nConfig` - when enabled, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. Otherwise, the `i18n` config should be passed directly to this package configuration. You can read more about it on [Internationalization](../advanced/internationalization.md) page. There are things specific to commercetools, which you can read more about below.
+
+### Adding states for the country
+In commercetools, each state must have a specified tax rule. In order to add a new state, add `states` field in the desired country inside `i18n.countries` in `nuxt.config.js`:
+```js
+i18n: {
+  countries: [
+    { name: 'US',
+      label: 'United States',
+      states: [
+        'California',
+        'Nevada'
+      ]
+    },
+    // ...
+  ]
+}
+```
+Then in commercetools' dashboard, open *Settings* -> *Project settings* -> *Taxes* and click **Add rate** button for new states:
```

Comment:
I will put whole part with tax rules as a toggle and just add warning that each state needs a tax rule. 


You describe it as it was a natural part of adding a new state to the config and you assumed that the tax rule is not assigned. I think there is a context mismatch VSF/CT configuration. It's good to mention that it's needed but it shouldn't interrupt the reading flow because you don't know how it was configured previously

Reply:
hmm, I suggest putting just the links to the Commercetools about the configuring states - that's it.

Reply:
Ok, I've put it in the toggle